draw-sine.lua

```diff
@@ -0,0 +1,51 @@
+-- Aseprite script to draw a sine wave with specified wave frequency and amplitude dampening
+-- Written by Dylmm, 2022
+-- https://github.com/Dylmm/aseprite-scripts
+
+-- Open dialog, ask user for paramters
+function userInput()
+    local dlg = Dialog()
+    -- Create dialog parameters
+    dlg:number{ id="wavefrequency", label="Wave Frequency:", decimals=2, text = "5"}
+    dlg:number{ id="wavedampening", label="Wave Dampening:", decimals=2, text = "1" }
+    dlg:check{ id="fill", label="Fill:", selected=true }
+    dlg:button{ id="ok", text="OK" }
+    dlg:button{ id="cancel", text="Cancel" }
+    dlg:show()
+
+    return dlg.data
+end
+
+-- Draws the specified sin wave
+function drawSine(wavefrequency,wavedampening,fill)
+    local image = app.activeCel.image
+    local copy = image:clone()
+    py = image.height / 2 
```

Comment:
It would be nice to specify an area to draw the sine wave in, rather than making it full screen. I think that would require x/y/width/height parameters though..